packages/utils/src/index.ts

```diff
@@ -1,6 +1,7 @@
 export { callAll } from './utils/CallAll';
 export { throttle } from './utils/throttle';
 export * as dateFormatters from './utils/dates/dateFormatters';
+export * as timeAgo from './utils/time/timeAgo';
```

Comment:
Please move all this code under `/utils/dates/`. I don't see any reason to create a separate directory for this.

Reply:
done